tests/enableCache/perspectives.js

```diff
@@ -188,8 +183,7 @@ describe(`tests/enableCache/perspectives.js, api: GET ${path} >`, () => {
           expect(jsonReply.name).to.equal(`${tu.namePrefix}testPersp`);
           expect(jsonReply.rootSubject).to.equal('myMainSubject');
           expect(jsonReply.lensId).to.equal(lensId);
-
-          done();
+          return done();
         } else {
           throw new Error('Expected response value in cache');
```

Comment:
our standard test semantics is to return `done(...)` with some value, e.g. `done('Expected response value in cache');`